kolibri/core/assets/src/views/sync/SelectSyncSourceModal.vue

```diff
@@ -85,7 +85,7 @@
       },
       localNetworkDescription: {
         message:
-          'Sync facility data with another instance of Kolibri on your local network or the internet',
+          'Sync facility data with another Kolibri server in your local network or the internet',
```

Comment:
I think "on a network" is more common than "in a network" https://ell.stackexchange.com/questions/40946/in-the-network-or-on-the-network

```suggestion
          'Sync facility data with another Kolibri server on your local network or the internet',
```